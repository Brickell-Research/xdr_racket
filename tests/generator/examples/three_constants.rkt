#lang xdr/generator

const FOO = 42
const BAR = 100.0
const BAZ = "hello"
