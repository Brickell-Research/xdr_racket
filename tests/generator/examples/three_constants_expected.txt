'(define FOO 42)
'(define BAR 100.0)
'(define BAZ "hello")
